src/terms/ewald.jl

```diff
@@ -33,31 +33,39 @@ function energy_ewald(model::Model{T}; kwargs...) where {T}
     energy_ewald(model.lattice, charges, model.positions; kwargs...)
 end
 
+# We balance the Ewald spling of ``1/r ≡ erf(η·r)/r + erfc(η·r)/r`` between reciprocal
```

Comment:
splitting

Reply:
Split that comment in two: first introduce the ewald splitting formula before the function definition (to explain what the function does), then explain the choice before the last line of the function (to explain how it does it)